.githooks/pre-commit

```diff
@@ -212,7 +212,7 @@ def check_spelling(filepath):
         return {}, [f"Error checking spelling: {str(e)}"]
 
 def update_dates():
-    """Update date fields in staged markdown files (date for new files, lastmod for modified)"""
+    """Update date fields in changed markdown files (date for new files, lastmod for modified)"""
```

Comment:
Does updating the dates for changed vs. staged files make a big difference for your use case?